tests/integration/test_database.py

```diff
@@ -44,53 +44,38 @@ async def test_database_relation(ops_test: OpsTest) -> None:
         ),
     )
 
-    mysql_app, mysqlrouter_app, application_app = applications[0], applications[1], applications[2]
+    mysql_app, application_app = applications[0], applications[2]
 
     await ops_test.model.relate(
-        f"{APPLICATION_APP_NAME}:database", f"{MYSQL_ROUTER_APP_NAME}:database"
+        f"{MYSQL_ROUTER_APP_NAME}:backend-database", f"{MYSQL_APP_NAME}:database"
```

Comment:
Am I correct in assuming that doing `backend-database:database` -> `database:database` was not previously possible? (inferring this from the PR title). And that this test covers the new behavior of relating the other way around?

If that is the case, I would also add a test (maybe a loop in this test?) testing the reverse, `database:database` -> `backend-database:database` too. I'm sure it's being tested elsewhere, but is useful to have both cases explicitly tested in the same place.

If I am misunderstanding, please ignore.

Reply:
You are correct, I have no tests for testing the reverse `database:database` -> `backend-database:database`. I'll open a separate PR to test this scenario